src/ralph/back_office/models.py

```diff
@@ -296,3 +298,17 @@ def return_report(self, request, **kwargs):
         self._generate_report(name='return', request=request)
     return_report.return_attachment = True
     return_report.verbose_name = _('Return report')
+
+
+@receiver(pre_save, sender=BackOfficeAsset)
+def hostname_assigning(sender, instance, raw, using, **kwargs):
+    """A hook for assigning ``hostname`` value when an
+    back office asset is edited.
```

Comment:
change this comment - hostname is assigned for new assets with in_progress status or edited assets when status has changed to in_progress.